app/__init__.py

```diff
@@ -1,18 +1,44 @@
-import os
-from flask import Flask, render_template, Response, request
+from datetime import timedelta
+from functools import wraps
+from flask import Flask, render_template, Response, request, redirect, url_for, session, flash
 from werkzeug.security import check_password_hash, generate_password_hash
 from . import db
 from app.db import get_db
+import os
 
 app = Flask(__name__)
 app.config['DATABASE'] = os.path.join(os.getcwd(), 'flask.sqlite')
+
+# Key for keeping client/server connection secure. 
+app.secret_key = 'd078e5e4bbd244e1a18ab1d1890157b7'
```

Comment:
Looks good but I think our `app.secret_key` shouldn't be visible to the public

Reply:
I'll look into that! 👍🏻